api/pkg/db/model/model.go

```diff
@@ -21,19 +21,19 @@ type (
 
 	Catalog struct {
 		gorm.Model
-		Name       string
-		Type       string
-		URL        string
-		Owner      string
+		Name       string `gorm:"size:100;not null;unique"`
```

Comment:
should we set a size?

Reply:
i am confused whether to set or size, by default it is 255